mobilecoind-json/tests/integration_test.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2022 The MobileCoin Foundation
+#
+# Integration test that uses mobilecoind-json to submit a transaction and check balance.
+
+import argparse
+import glob
+import json
+import logging
+import os
+import sys
+import time
+import urllib.request
```

Comment:
I know `requests` exists and is more pleasant to use but I wanted something that has no external dependencies.